src/helpers/mixed.js

```diff
@@ -152,3 +152,19 @@ function _checkKeySchema(v) {
   return p === z;
 }
 /* eslint-enable */
+
+/**
+ * Cross-platform helper to clear text selection.
+ */
+export function clearTextSelection() {
```

Comment:
The better place for this function will be the `dom/element.js` file. There you can find functions similar to this one, such as `getSelectionText` (https://github.com/handsontable/handsontable/blob/master/src/helpers/dom/element.js#L851)